Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -1725,6 +1742,10 @@ protected function forgetChangedClasses(): void
 
     /**
      * Forgets all reflection data related to the specified class
+     *
+     * @param (int|string) $className
+     *
+     * @psalm-param array-key $className
```

Comment:
Huh? Why `array-key`?

Reply:
No idea, Psalm added it. I'll go through all doc comments and clean up in a separate PR.